### DISCUSSION TITLE
Implement wrong keyboard layout interpretation for search bars

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#26373] [Ideas]
The osu! community is multinational so multilingual. Most of individuals use at least 2 keyboard layouts. There were so many times when I tried to search something but constantly made request on Russian layout like "Кфшыу ьн" -> "Raise my", "skin" -> "ылшт", "fps" -> "азы" and have gotten nothing. 
Would be cool if osu! had this "smart" algorithm that would give searching results even with wrong keyboard layout search input.

**Comment by bdach** [comment]
Not happening. It would be annoying to implement and also slow and search has performance issues anyway.
